src/platform/packages/shared/kbn-esql-ast/src/definitions/utils/expressions.test.ts

```diff
@@ -303,10 +306,6 @@ describe('getExpressionType', () => {
         expression: '[1., 2.]',
         expectedType: 'double',
       },
-      {
-        expression: '[null, null, null]',
-        expectedType: 'null',
-      },
```

Comment:
Why this removal?

Reply:
Because `[null, null, null]` does not exist in ES|QL, it does not have lists of nulls, AFAIK. My understanding is that this test does not test what it intends, it was just an accident that it worked.